### PR TITLE
Add console task

### PR DESCRIPTION
**Pull request by dolzenko**

### CODE REVIEW

**Comment by dim** [comment]
I wonder if it makes more sense to add a `grape-app c[onsole] ENV` command to mimic rails

**Comment by dolzenko** [comment]
Yeah, makes sense, will add.

**Comment by mxmCherry** [comment]
This is a nice feature, seems to have been forgotten?

Would be also nice to have similar task in Rakefile, if it's not generated yet.

File: lib/grape/app/cli.rb [hunk 1]
```diff
@@ -30,6 +30,15 @@ def init_lib
 
   class Runner < Thor
     register Builder, :new, "new NAME", "create a new application"
+
+    desc 'console ENV', 'Runs console'
+    def console(env = 'development')
+      ENV['GRAPE_ENV'] = env
+      require File.expand_path('config/environment', Dir.pwd)
+      require 'irb'
```
Comment (dim) [inline lib/grape/app/cli.rb:38]:
```
require 'irb'
require 'irb/completion'
```

**Comment by dim** [comment]
Declining in favour of #8

**Pull request closed without merging**